data/json/terrain.json

```diff
@@ -3156,9 +3156,29 @@
         "id" : "t_tree_maple",
         "name": "maple tree",
         "symbol": "7",
-        "color": "red",
+        "color": [ "light_green", "green", "red", "brown" ],
```

Comment:
The colors aren't just there for decoration but more importantly to mark when is the tree harvestable and when it isn't. Green is for trees that won't become harvestable, light green for ones that are not harvestable at the moment (because they were harvested recently or because the season is wrong), brown is currently used for all trees in winter.